### PR TITLE
After drag off the screen horizontally close the floating view using raw pip view

### PR DESCRIPTION
**Pull request by dinakar-m**
Feature -
After drag off the screen horizontally close the floating view using raw pip view.

Android phone video - 

https://user-images.githubusercontent.com/102947879/181443146-1e9fa714-c93e-4488-aa1c-af1b4b25a0b4.mov

Android phone screens -

Screenshot_1658991902
Screenshot_1658991899
Screenshot_1658991858

iOS
Simulator Screen Shot - iPod touch (7th generation) - 2022-07-28 at 16 06 57
Simulator Screen Shot - iPod touch (7th generation) - 2022-07-28 at 16 07 01
Simulator Screen Shot - iPod touch (7th generation) - 2022-07-28 at 16 07 25

### CODE REVIEW

File: lib/src/raw_pip_view.dart [hunk 1]
```diff
@@ -149,8 +173,8 @@ class RawPIPViewState extends State<RawPIPView> with TickerProviderStateMixin {
         final bottomWidget = widget.bottomWidget ?? _bottomWidgetGhost;
         final width = constraints.maxWidth;
         final height = constraints.maxHeight;
-        double? floatingWidth = widget.floatingWidth;
-        double? floatingHeight = widget.floatingHeight;
+        var floatingWidth = widget.floatingWidth;
+        var floatingHeight = widget.floatingHeight;
```
Comment (enricmacias) [inline lib/src/raw_pip_view.dart:177]: Where is the need to change this? Can we keep the `double` ? But without the optional

File: lib/src/raw_pip_view.dart [hunk 2]
```diff
@@ -333,7 +357,7 @@ Map<PIPViewCorner, Offset> _calculateOffsets({
   }
 
   final corners = PIPViewCorner.values;
-  final Map<PIPViewCorner, Offset> offsets = {};
+  final offsets = <PIPViewCorner, Offset>{};
```
Comment (enricmacias) [inline lib/src/raw_pip_view.dart:360]: Why is this changed too? It is not the same?

**Review by enricmacias — APPROVED** [review]
@dinakar-m 
No need to change the `PIPView` class? Only the `RawPIPView` ?

File: lib/src/raw_pip_view.dart (hunk 2 above)
Comment (dinakar-m) [reply lib/src/raw_pip_view.dart]: both data types changes i did when i copy pasted the code in example app
but in this case dont need.
so i kept original.

**Comment by dinakar-m** [comment]
> @dinakar-m No need to change the `PIPView` class? Only the `RawPIPView` ?

i did not changed the default function of pip view.

pipview -> basic default uses
RawPIPView -> use for advance function like on tap close and get call back, drag off etc.

this time we are more interested in raw pip view so i thought better to modify RawPIPView only.
